xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -29,18 +29,31 @@
 ## Title and parent
 ##
 <div id="parentinput" class="form-group">
-  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
+  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') 
+  <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
```

Comment:
The ``a`` element is not on the same level as the ``label``. In fact it is a child element, and child elements should be indented.

Reply:
Alright, sorry I didn't notice!